docs/commands.md

```diff
@@ -133,13 +133,13 @@ If this command is used to add data to an existing timeseries, `retentionSecs` a
 Create a compaction rule.
 
 ```sql
-TS.CREATERULE sourceKey destKey AGGREGATION aggType bucketSizeSeconds
+TS.CREATERULE sourceKey destKey AGGREGATION aggType bucketSize
```

Comment:
`bucketByTime` might be more representative. Size usually refers to memory

Reply:
maybe we should change it to just "interval"?

Reply:
@gkorland do you think its representative of the meaning? its more a time-bucket ("truncated time").
I think we can go simple and just call this `timeBucket`

Reply:
done